### PR TITLE
Platform configuration guide for Bare Metal.

### PR DESCRIPTION
**Pull request by ipochi**
Signed-off-by: Imran Pochi <imran@kinvolk.io>

### CODE REVIEW

**Comment by ipochi** [comment]
@iaguis 

Created for bare metal. This one doesnt have the installating/uninstalling and prerequisites sections.

Please let me know if you think this makes more sense or having the above sections as they were for platform and aws

File: docs/configuration-guides/platforms/baremetal.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# Lokomotive Bare Metal Installation Guide
```
Comment (iaguis) [inline docs/configuration-guides/platforms/baremetal.md]:
```suggestion
# Lokomotive bare metal installation guide
```

File: docs/configuration-guides/platforms/baremetal.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+|-----------------------------|--------------------------------------------------------------|:-------:|:--------:|
+| `backend.local`             | Local backend configuration block.                           | -       | false    |
+| `backend.local.path`        | Location where Lokomotive stores the cluster state.          | -       | false    |
+| `backend.s3`                | AWS S3 backend configuration block.                          | -       | false    |
+| `backend.s3.bucket`         | Name of the S3 bucket where Lokomotive stores cluster state. | -       | true     |
+| `backend.s3.key`            | Path in the S3 bucket to store the cluster state.            | -       | true     |
+| `backend.s3.region`         | AWS Region of the S3 bucket.                                 | -       | false    |
+| `backend.s3.aws_creds_path` | Path to the AWS credentials file.                            | -       | false    |
+| `backend.s3.dynamodb_table` | Name of the DynamoDB table for locking the cluster state.    | -       | false    |
+
```
Comment (iaguis) [inline docs/configuration-guides/platforms/baremetal.md]:
```suggestion
```

**Comment by ipochi** [comment]
@iaguis I like this one better without the sections of (Installing, uninstalling and prerequisites)
What is your opinion ? I will update the other platform guides to remove the sections if you feel the same as I do.

**Pull request merged**